Source/IInterceptStrategy.cs

```diff
@@ -142,7 +138,7 @@ internal IEnumerable<IProxyCall> OrderedCalls
 			{
 				lock (orderedCalls)
 				{
-					return orderedCalls.ToList();
+					return orderedCalls.ToArray();
```

Comment:
Micro-optimization. Since callers are only going to read the returned collection, choose a type with as little overhead as possible.